### PR TITLE
open unit test for bf16

### PR DESCRIPTION
**Pull request by asleepzzz**
due to we can use bf16 after merged pr:
https://github.com/ROCmSoftwarePlatform/llvm-project-mlir/pull/85
we open bf16 unit test again

### CODE REVIEW

**Review by whchung — APPROVED** [review]
@asleepzzz Thanks. Please help open another PR for fp16. And let's monitor CI results.

File: mlir/test/mlir-miopen-driver/sanity_pv_xdlops.mlir [hunk 1]
```diff
@@ -5,4 +5,4 @@
 // FIXME: investigate why these commands fail on non-gfx908 CI nodes.
```
Comment (whchung) [inline mlir/test/mlir-miopen-driver/sanity_pv_xdlops.mlir]: remove FIXME and UNSUPPORTED

**Review by whchung — CHANGES_REQUESTED** [review]

**Comment by whchung** [comment]
@asleepzzz This test crashed on CI:

```
mlir-miopen-driver -p -t bf16 -x2 -miopen-lowering -miopen-affine-transform -miopen-affix-params -miopen-lowering-step2 -miopen-lowering-step3 -miopen-lowering-step4
```

Please help look into why. I was puzzled by this a couple of days ago too.

**Comment by whchung** [comment]
CI log:
http://ml-ci.amd.com:21096/job/MLIR/job/MLIR-downstream-unit-test/2957/console

**Comment by asleepzzz** [comment]
When PR#90 merged, we can merge this PR

**Comment by whchung** [comment]
jenkins: retest this please.

**Comment by whchung** [comment]
@asleepzzz Please help address review comments.

File: mlir/test/mlir-miopen-driver/sanity_pv_xdlops.mlir (hunk 1 above)
Comment (asleepzzz) [reply mlir/test/mlir-miopen-driver/sanity_pv_xdlops.mlir]: remove it and change to run

Comment (asleepzzz) [reply mlir/test/mlir-miopen-driver/sanity_pv_xdlops.mlir]: remove comments

File: mlir/test/mlir-miopen-driver/sanity_pv_xdlops.mlir [hunk 2]
```diff
@@ -2,7 +2,6 @@
 // Sanity test to -pv produces valid MLIR for all possible data types and
```
Comment (whchung) [inline mlir/test/mlir-miopen-driver/sanity_pv_xdlops.mlir:1]: @asleepzzz remove `UNSUPPORTED` clause. This test is NOT run right now.

Comment (asleepzzz) [reply mlir/test/mlir-miopen-driver/sanity_pv_xdlops.mlir:1]: remove UNSUPPORTED

**Review by whchung — APPROVED** [review]

**Pull request merged**